hedera-node/data/config/bootstrap.properties

```diff
@@ -10,7 +10,3 @@ accounts.storeOnDisk=false
 tokens.storeRelsOnDisk=false
 tokens.nfts.useVirtualMerkle=false
 cache.cryptoTransfer.warmThreads=30
-records.useConsolidatedFcq=true
```

Comment:
Are we certain these can be removed?  Might want to check with Platform if we haven't already done so.

Reply:
Actually, it appears this is a merge artifact, it was merged to develop in PR 6781.  Odd that something re-ran that commit on this branch.